soc/bflb/bl70x/Kconfig.defconfig

```diff
@@ -5,9 +5,9 @@
 if SOC_SERIES_BL70X
 
 # On BL702, BL704 and BL706, not all values work, here is a list of some that work:
-# 2000, 1000, 100, 10, 1
+# 1000, 100, 10, 1
```

Comment:
Why are we changing this ? Another commit with clear explanation.

Reply:
> it seems that you fix the problem but it was a guess because the technical details are not present

I thought I made it obvious enough this was the case here. It fixes the issue, which is critical, and I dont know why.